src/settings.rs

```diff
@@ -281,6 +284,12 @@ impl Settings {
         settings = settings.add_source(app_env);
 
         // add custom override
+        if let Ok(openai_api_base) = std::env::var("OPENAI_API_BASE") {
```

Comment:
I assume this ENVVAR is pretty standard across OpenAI tools. Looks good to me

Reply:
ok, I'll do it